templates/tpos/tpos.html

```diff
@@ -996,7 +1021,12 @@ <h5 class="q-mt-none q-mb-sm">
       },
       submitForm: function () {
         if (this.total != 0.0) {
-          this.stack = Array.from(String(Math.ceil(this.total * 100)), Number)
+          this.stack = Array.from(
+            String(
+              Math.ceil(this.total * (this.currency == 'satoshi' ? 1 : 100))
+            ),
+            Number
+          )
```

Comment:
I think this is easier to read:

```suggestion
          if (this.currency == 'satoshi') {
              this.stack = Array.from(String(Math.ceil(this.total), Number)
          } else {
              this.stack = Array.from(String(Math.ceil(this.total * 100)), Number)
          }
```